Source/WebKit/UIProcess/API/APIContentRuleListStore.cpp

```diff
@@ -234,6 +236,19 @@ static std::optional<MappedData> openAndMapContentRuleList(const WTF::String& pa
     auto metaData = decodeContentRuleListMetaData(fileData);
     if (!metaData)
         return std::nullopt;
+    const size_t headerAndSourceSize = headerSize(metaData->version) + metaData->sourceSize;
```

Comment:
nit: I'd add a newline before this.  Actually, I'd put all this into a helper function.